ansible_rulebook/util.py

```diff
@@ -202,3 +203,8 @@ def check_jvm():
             file=sys.stderr,
         )
         sys.exit(1)
+
+
+def untar(filename: str, output_dir: str) -> None:
```

Comment:
why lose the asynchrony?

Reply:
@Alex-Izquierdo Not sure I haven't found a package that uses aiofile and tar file to do async processing. The earlier implementation was just using the asyncio.create_subprocess_exec to run the tar command directly.

Reply:
@Alex-Izquierdo I reverted to the old create_subprocess_exec.